Jenkinsfile.cd

```diff
@@ -92,6 +95,24 @@ def notifyingFailure() {
     }
 }
 
+def staticValidation() {
+    parallel([
+        'libindy'    : { runValdiationRust('libindy') },
+        'libnullpay' : { runValdiationRust('libnullpay') },
+        'libvcx'     : { runValdiationRust('vcx/libvcx') },
+        'cli'        : { runValdiationRust('cli') }
+    ])
+}
+
+def runValdiationRust(dir) {
+    node('ubuntu') {
+        def testEnv = buildTestEnv("ci/static-validation.dockerfile ci", "static-validation-ubuntu")
```

Comment:
- there is no file `ci/static-validation.dockerfile` in indy-sdk repo
- add `checkout scm` before building test env
- use this instead `def testEnv = dockerBuild('static-validation-ubuntu', "ci/static-validation.dockerfile ci")`

Reply:
Wow, I missed a bunch of stuff, haha. Everything should be fixed now